simdv2.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -x
+
+rm -rf $HOME/.simapp
+
+ROOT=$PWD
+SIMD="${SIMD:-go run ./simd/main.go}"
+CONFIG="${CONFIG:-$HOME/.simapp/config}"
+
+cd simapp
+$SIMD init aurn-node --chain-id aurn-chain
+
+cd $CONFIG
+
+# to enable the api server
+# sed -i '.bak' '/\[api\]/,+3 s/enable = false/enable = true/' app.toml
+sed -i -E 's|enable = false|enable = true|g' app.toml
+
+# to change the voting_period
+jq '.app_state.gov.voting_params.voting_period = "600s"' genesis.json > temp.json && mv temp.json genesis.json
+
+# to change the inflation
+jq '.app_state.mint.minter.inflation = "0.300000000000000000"' genesis.json > temp.json && mv temp.json genesis.json
+
+cd "$ROOT"/simapp
+$SIMD keys add test_validator --keyring-backend test
+VALIDATOR_ADDRESS=$($SIMD keys show test_validator -a --keyring-backend test)
+
+$SIMD genesis add-genesis-account "$VALIDATOR_ADDRESS" 1000000000stake
+$SIMD genesis gentx test_validator 1000000000stake --keyring-backend test
+$SIMD genesis collect-gentxs
+
+cd v2/simdv2/
+go run main.go start
```

Comment:
This is not doing a lot. Also bash can be very annoying. I would be happy to help migrating this to system tests if you can outline the requirements or test assumptions. 
When I was testing with app v2, I had some problems on chain upgrade but CI is prepared: https://github.com/cosmos/cosmos-sdk/blob/main/.github/workflows/test.yml#L190

Reply:
I need simapp v2s init and genesis code paths tested so that we don't get regressions in this branch. roughly

```bash
$SIMD init test-node --chain-id test-chain
$SIMD genesis add-genesis-account "$VALIDATOR_ADDRESS" 1000000000stake
$SIMD genesis gentx test_validator 1000000000stake --keyring-backend test
$SIMD genesis collect-gentxs
# edit config yaml to halt after 5 blocks
$SIMD start 
# assert exit status 0
```